test/testConstants/BusinessDetailsTestConstants.scala

```diff
@@ -39,8 +39,10 @@ object BusinessDetailsTestConstants {
   val address = AddressModel("8 Test", Some("New Court"), Some("New Town"), Some("New City"), Some("NE12 6CI"), "United Kingdom")
   val businessIncomeSourceId = "1234"
 
+  val fixedDate : LocalDate = LocalDate.of(2023, 12, 15)
+
   val getCurrentTaxYearEnd: LocalDate = {
-    val currentDate: LocalDate = LocalDate.now
+    val currentDate: LocalDate = fixedDate
```

Comment:
:wrench: perhaps rename this as it's not longer the current date

Reply:
fixed

Reply:
uhh it's still called *currentDate* for me

Reply:
looks like some gitHub glitch, check one more time